src/System.IO.Compression.ZipFile/pkg/System.IO.Compression.ZipFile.pkgproj

```diff
@@ -2,18 +2,36 @@
 <Project ToolsVersion="12.0" DefaultTargets="Build" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
   <Import Project="$([MSBuild]::GetDirectoryNameOfFileAbove($(MSBuildThisFileDirectory), dir.props))\dir.props" />
   <ItemGroup>
+    <ProjectReference Include="..\ref\4.0.0\System.IO.Compression.ZipFile.depproj">
+      <SupportedFramework>net45;netcore45;wpa81</SupportedFramework>
+    </ProjectReference>
     <ProjectReference Include="..\ref\System.IO.Compression.ZipFile.csproj">
       <SupportedFramework>net46;netcore50;netstandardapp1.5</SupportedFramework>
     </ProjectReference>
     <ProjectReference Include="..\src\System.IO.Compression.ZipFile.builds" />
```

Comment:
Not shown in this diff, but changing the configuration of this builds and the CSProj it points at would allow you to define more configurations to cross compile for.  That could get you close to supporting netstandard1.1 but I think we'd be blocked by lack of FileSystem contract support on win8-8.1 and wpa8.1.